test/unit/view_carbon_tax_calculation_test.sql

```diff
@@ -193,20 +273,19 @@ select results_eq(
           on fm.fuel_carbon_tax_details_id = ctd.id
           join swrs.carbon_tax_act_fuel_type cta
           on ctd.carbon_tax_act_fuel_type_id = cta.id
-          join swrs.fuel_charge fc
-          on fc.carbon_tax_act_fuel_type_id = cta.id
-          and concat(r.reporting_period_duration::text, '-12-31')::date between fc.start_date and fc.end_date
+          where e.emission_category = 'BC_ScheduleB_FlaringEmissions'
       )
-      select round(x.fuel_charge * x.natural_gas_amount, 2) from x;
+      select x.natural_gas_amount from x
     $$,
 
-    'swrs.carbon_tax_calculation properly calculates carbon tax based on flaring emission quantity'
+    'swrs.carbon_tax_calculation properly converts flaring emissions to a natural gas fuel amount'
 );
 
 select results_eq(
     $$
       select fuel_amount from swrs.carbon_tax_calculation
-      where fuel_type='Natural Gas (Sm^3)';
+      where fuel_type='Natural Gas (Sm^3)'
+      and emission_category='BC_ScheduleB_VentedEmissions';
```

Comment:
```suggestion
      and emission_category='BC_ScheduleB_VentingEmissions';
```

Reply:
It is Vented or Venting? I guess it's the same comment as the carbon tax calculation tests above, in that they're quite redundant, and if you make the same mistake in both "expected" and "actual" queries then you're not actually testing much. The second part of this test should just be `10 * (1000000 / 678.5)` instead of a big query?

This applies to the flaring test above too